sdk/framework/backend.go

```diff
@@ -83,6 +83,10 @@ type Backend struct {
 	// Type is the logical.BackendType for the backend implementation
 	BackendType logical.BackendType
 
+	// Initialize is invoked just after mounting a backend to allow it to
+	// handle any initialization tasks that need to be performed.
+	InitializeFunc InitializeFunc
```

Comment:
Is there a reason not to just call it Initialize instead of InitializeFunc (e.g. how Invalidate is an InvalidateFunc)

Reply:
Agreed that `Initialize` would be better, and thats what I named it at first, but the compiler doesn't like it: 

```
sdk/framework/backend.go:275:6: type Backend has both field and method named Initialize
```